test/junit/scala/collection/immutable/StringLikeTest.scala

```diff
@@ -1,104 +0,0 @@
-package scala.collection.immutable
-
-import org.junit.Assert._
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.JUnit4
-
-import scala.tools.testkit.AssertUtil._
-import scala.util.Random
-
-/* Test for scala/bug#8988 */
-@RunWith(classOf[JUnit4])
-class StringLikeTest {
```

Comment:
Ditto; this tests `StringOps` now, presumably. Maybe rename it?

Reply:
Yes, it's just a name. My comment on the ticket was that the spec needs fixing, maybe I forgot to say the tests are fine. If they actually referenced a missing class, they wouldn't compile. Does naming even matter? They are testing behavior, not names.